### PR TITLE
Add openQA_worker container test

### PR DESCRIPTION
**Pull request by ilausuch**
Once a single webUI container is working, this test creates a
worker container and check if connects to the web UI.

https://progress.opensuse.org/issues/80682

### CODE REVIEW

File: tests/containers/worker.pm [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  my ($self) = @_;
+  my $volumes = "-v \"/root/data/factory:/data/factory\" -v \"/root/data/tests:/data/tests\" -v \"/root/openQA/container/webui/conf:/data/conf:ro\"";
+
+  assert_script_run("cat '127.0.0.1 openqa_webui' >>/etc/hosts");
+  assert_script_run("echo -e '[openqa_webui]\nkey=1234567890ABCDEF\nsecret=1234567890ABCDEF' >/data/conf/client.conf");
+
+  assert_script_run("docker run --rm -d --network testing $volumes --name openqa_worker openqa_worker");
+
+  wait_for_container_log("openqa_worker", "Registering with openQA", "docker");
+  record_info("worker inizializated");
```
Comment (okurz) [inline tests/containers/worker.pm:15]: typo in the string. Better just delete this line :)

File: tests/containers/worker.pm [hunk 2]
```diff
@@ -0,0 +1,19 @@
+use Mojo::Base qw(openQAcoretest);
+use testapi;
+use utils;
+
+sub run {
+  my ($self) = @_;
+  my $volumes = "-v \"/root/data/factory:/data/factory\" -v \"/root/data/tests:/data/tests\" -v \"/root/openQA/container/webui/conf:/data/conf:ro\"";
```
Comment (okurz) [inline tests/containers/worker.pm:7]: Please use single-ticks by default unless you want to expand perl variables within. Then you would not need the additional quote escaping.

**Review by okurz — CHANGES_REQUESTED** [review]

File: tests/containers/worker.pm (hunk 1 above)
Comment (ilausuch) [reply tests/containers/worker.pm]: Done

File: tests/containers/worker.pm (hunk 2 above)
Comment (ilausuch) [reply tests/containers/worker.pm]: Done

Comment (okurz) [reply tests/containers/worker.pm]: well, it was meant as a generic suggestion, not just for this line :)

Please do the same for the other statements as well.

Comment (ilausuch) [reply tests/containers/worker.pm]: Ok, done

**Comment by Martchus** [comment]
Note that GitHub fails to associate the commit with your use account. That doesn't prevent us to merge it but maybe you want to fix it nevertheless.

File: tests/containers/worker.pm (hunk 2 above)
Comment (okurz) [reply tests/containers/worker.pm]: please also change https://github.com/os-autoinst/os-autoinst-distri-openQA/pull/61/files#diff-fba7d5b35a7ae1e16e0a7fd11bbcf260d1aa87d9323dd92eaaf0e300a5a799e5R10

**Comment by kalikiana** [comment]
Do you have a verification run?

**Comment by ilausuch** [comment]
Yes, sorry
https://openqa.opensuse.org/tests/1613965

**Review by kalikiana — APPROVED** [review]

**Pull request merged**